esphome/components/ade7953_i2c/sensor.py

```diff
@@ -0,0 +1,27 @@
+import esphome.codegen as cg
+import esphome.config_validation as cv
+from esphome.components import i2c, ade7953_base
+from esphome.const import CONF_ID
+
+
+DEPENDENCIES = ["i2c"]
+AUTO_LOAD = ["ade7953_base"]
+
+ade7953_ns = cg.esphome_ns.namespace("ade7953_i2c")
+ADE7953 = ade7953_ns.class_("AdE7953I2c", cg.PollingComponent, i2c.I2CDevice)
```

Comment:
This should be showing that is extends `ade7953_base::ADE7953`

The class stub should be created in the _base python file and imported here.

```suggestion
ADE7953 = ade7953_ns.class_("AdE7953I2c", ade7953_base.ADE7953, i2c.I2CDevice)
```